framework/web/UrlManager.php

```diff
@@ -488,4 +488,16 @@ public function setHostInfo($value)
     {
         $this->_hostInfo = rtrim($value, '/');
     }
+
+    /**
+     * Remove numeric indexes from the response
+     * @param $params
+     *
+     * @return mixed|string
+     */
+    protected function custom_http_build_query($params) {
+        $request = http_build_query($params);
+        $request = preg_replace('#%5B[0-9]%5D#', '%5B%5D', $request);
```

Comment:
@nineinchnick I forgot the + from the [0-9]+. What do you mean with break the param values? I'll make unit test if I understand what could be broken